README.md

```diff
@@ -166,6 +166,10 @@ Schemer\Validator::boolean() // Accept only boolean values.
     ->true() // Accept only 'true'.
     ->false() // Accept only 'false'.
 
+Schemer\Validator::boolean("George says no") // Accept only boolean values, with a custom error.
```

Comment:
I'm worried about this making the README too bloated; maybe we'll go for a note at the top ("all these take a custom $error string" etc) and then just an `$error` param in all the functions already there? What do you think?